common/test/acceptance/pages/studio/course_info.py

```diff
@@ -1,16 +1,150 @@
 """
 Course Updates page.
 """
-
+from common.test.acceptance.pages.common.utils import click_css, confirm_prompt
 from common.test.acceptance.pages.studio.course_page import CoursePage
+from common.test.acceptance.pages.studio.utils import type_in_codemirror, set_input_value
 
 
 class CourseUpdatesPage(CoursePage):
     """
     Course Updates page.
     """
-
     url_path = "course_info"
 
     def is_browser_on_page(self):
-        return self.q(css='body.view-updates').present
+        """
+        Returns whether or not the browser on the page and has loaded the required content
+
+        Check for the presence of handouts-content, when it is present the render function has completed
```

Comment:
Nit: I meant for you to put this comment before the line of code, not in the "public" API documentation section. It's a detail that isn't necessary for the caller.

Also, in general I am a fan of punctuation (periods and question marks).